cssdb.json

```
@@ -59,6 +59,15 @@
       }
     ]
   },
+  {
+    "id": "color-adjust",
+    "title": "`color-adjust` Property",
+    "description": "The color-adjust property is a non-standard CSS extension that can be used to force printing of background colors and images",
+    "specification": "https://drafts.csswg.org/css-color/#color-adjust",
+    "stage": 1,
```

This might be Stage 2, because it has advanced as a Working Draft to w3.org. https://cssdb.org/#stage-2

```suggestion
    "stage": 2,
```